schema/json.go

```diff
@@ -51,12 +51,20 @@ func (dst *JSON) Set(src interface{}) error {
 
 	switch value := src.(type) {
 	case string:
-		*dst = JSON{Bytes: []byte(value), Status: Present}
+		if value == "" {
+			*dst = JSON{Bytes: []byte(`""`), Status: Present}
```

Comment:
I believe `JSON{Status: Null}` would be more correct here.
Current implementation results in a `""` value in the postgres column (which is kinda weird - because likely the policy is an object of some-sort like `{"some-policy": "some-value"}`).

Just null i think - no reason to distinguish between empty-string and nonexistent.

Reply:
I was trying to keep the same behavior that go has when we call `Marshal`...

`json.Marshal("")` produces `""`